dist/pom.xml

```diff
@@ -177,6 +189,71 @@
 
     <profiles>
         <profile>
+            <id>integration</id>
+            <build>
+                <resources>
+                    <resource>
+                        <directory>${project.basedir}/src/main/assembly/conf</directory>
+                        <includes>
+                            <include>*.pem</include>
+                        </includes>
+                    </resource>
+                </resources>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-antrun-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>process-resources</id>
+                                <phase>process-resources</phase>
+                                <configuration>
+                                    <target>
+                                        <chmod file="${project.build.outputDirectory}/brooklyn.properties" perm="600"/>
+                                    </target>
+                                </configuration>
+                                <goals>
+                                    <goal>run</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>io.brooklyn.maven</groupId>
+                        <artifactId>brooklyn-maven-plugin</artifactId>
+                        <version>0.3.0-SNAPSHOT</version>
+                        <executions>
+                            <execution>
+                                <id>Start and deploy Clocker</id>
+                                <goals>
+                                    <goal>start-server</goal>
+                                    <goal>deploy</goal>
+                                    <goal>sensor</goal>
+                                    <goal>stop</goal>
+                                </goals>
+                                <configuration>
+                                    <mainClass>brooklyn.clocker.Main</mainClass>
+                                    <bindPort>${bindPort}</bindPort>
+                                    <pollPeriod>${brookyn.pollPeriod}</pollPeriod>
+                                    <timeout>30</timeout><!--minutes-->
+                                    <serverClasspathScope>runtime</serverClasspathScope>
+                                    <bindAddress>localhost</bindAddress>
+                                    <arguments>
+                                        <argument>--catalogInitial</argument>
+                                        <argument>${project.basedir}/src/main/assembly/files/clocker.bom</argument>
```

Comment:
Why not use `dist/src/main/assembly/files/conf/brooklyn/default.catalog.bom` instead?